### DISCUSSION TITLE
When are we getting 3 dans and above?

### DISCUSSION THREAD

**Discussion by njc-hellothisisme** [codewars/codewars.com#3138] [Q&A] [answered]
It's on the title. The docs says the dans go up to 8th dan, but they've only implemented 2nd dan.

**Comment by Blind4Basics** [comment, marked as answer]
all ranks are implemented, but there aren't enough kata for anyone to reach the rank threshold for 3 dan. It's even doubtful it will ever be  achievable (unless waiting for some decades, maybe)

**Reply by njc-hellothisisme** [reply]
Welp, no choice but to wait for harder questions then - I haven't seen any 1 dan questions. Thank you for your answer!

**Reply by hobovsky** [reply]
Why would we need them? Did you already complete all 1 kyu kata?  :)

**Reply by njc-hellothisisme** [reply]
No (I'm 6 kyu lol), but it'd be nice to see what the community comes up with, just for funsies :)
